service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -70,9 +71,9 @@ public void removePastHearings(List<Element<HearingBooking>> hearingDetails) {
         }
     }
 
-    public HearingBooking getMostUrgentHearingBooking(List<Element<HearingBooking>> hearingDetails) {
+    public static HearingBooking getMostUrgentHearingBooking(List<Element<HearingBooking>> hearingDetails) {
         return unwrapElements(hearingDetails).stream()
-            .filter(hearing -> hearing.getStartDate().isAfter(time.now()))
+            .filter(hearing -> hearing.getStartDate().isAfter(LocalDateTime.now()))
```

Comment:
Not sure what's exact rationale for this change (probably some difficulties with dependency injection), but technically this method is now impossible to test properly. 

I wonder how it's tested now.

Reply:
Ok, I see how it works, as we are operating on days from the test purposes, that change is unsafe though, if someone decides to use seconds differences in their test setup, this method will randomly fail.

Please consider passing the time as param then, or revert back the change.

Reply:
I am happy to explain about why this is anti pattern if you are not convinced :)